feat: add debug logging (#198)
fauna/fauna-python: README.md

client.change_feed(fql('Product.all().toStream()'), options)
```

## Logging

> Debug logging is handled by the standard logging package in under the `fauna` namespace. We will log the request with body, excluding the Authorization header, as well as the full response.

Reviewer: typo: `in under`

The Python's 
```suggestion
Logging is handled using Python's standard `logging` package under the `fauna` namespace. Logs include the HTTP request with body (excluding the `Authorization` header) and the full HTTP response.
```

> In your application, you can enable debug logging with the following. Given this is a standard convention in Python, you may want more fine-grained control over which libraries log. See Python's how-to at https://docs.python.org/3/howto/logging.html.

Reviewer: ```suggestion
To enable logging:
```

I'd move the info about configuring the logs after the example.

```python
import logging
from fauna.client import Client
from fauna import fql

logging.basicConfig(
    level=logging.DEBUG
)

> c = Client()
> c.query(fql("42"))

Reviewer: Super minor nits — just for consistency with earlier examples:

```suggestion
client = Client()
client.query(fql('42'))
```

```

> 

Reviewer: ```suggestion
For configuration options or to set specific log levels, see Python's [Logging HOWTO](https://docs.python.org/3/howto/logging.html).

```
